app/assets/javascripts/index.js.erb

```diff
@@ -462,6 +524,9 @@ class BirdMap {
 
 		const birds = await this.getBirds();
 		this.birds.insert(...birds);
+
+		const locations2 = await this.getLocations();
+		this.locations.insert(...locations2);

```

Comment:
This should just be `locations`, not `locations2`. There is a typo on line 523 that should be corrected as well; `this.locations` should be `this.hotspots`.